Utilities/lockless.h

```diff
@@ -390,9 +390,19 @@ class lf_queue final
 			item->m_link = load(oldv);
 		}
 
-		if (!oldv)
+		if (!oldv && Notify)
```

Comment:
Wouldn't this reduce instructions if it was behind if constexpr Notify?

Reply:
Not really, compilers detect that Notify is known. if-constexpr is not technically for optimizations, but to avoid compiling code that would otherwise not compile for template functions.